ansible/templates/usb/scripts/prompt-config.sh.j2

```diff
@@ -1476,11 +1569,17 @@ you use to create virtual machines through Danube Cloud.\n\n"
 		printf "$message"
 	fi
 
-	prompt_host_ok_val "Enter the Primary DNS server IP" "$dns_resolver1" "dns_resolver1"
-	dns_resolver1="$val"
-	prompt_host_ok_val "Enter the Secondary DNS server IP" "$dns_resolver2" "dns_resolver2"
-	dns_resolver2="$val"
-	promptval "Default DNS search domain" "$dns_domain" "dns_search"
+	if [[ $ADVANCED_INSTALL -eq 1 ]]; then
```

Comment:
Yes, but [this](https://github.com/erigones/esdc-factory/blob/167a68bfcf79610240b188165aa26930e6c9b4e3/ansible/templates/usb/scripts/prompt-config.sh.j2#L1666) also needed to change.